src/client.rs

```diff
@@ -69,10 +81,7 @@ impl Client {
         let url = self.build_url("ping", None);
         self.client.get(url).send().map(move |res| {
             if let Ok(res) = res {
-                match res.status().as_u16() {
```

Comment:
(again, I'm learning Rust, so sorry if this is annoying...)

What does this change do?  Is it just gardening, or is there a real, and required change in logic?

Reply:
It’s just clippy’s suggestion, no actual logical modification

Reply:
Awesome, thanks.